e2e/process-services-cloud/form-field/visibility-condition.e2e.ts

```diff
@@ -22,6 +22,7 @@ import { AlfrescoApiCompatibility as AlfrescoApi } from '@alfresco/js-api';
 import { NavigationBarPage } from '../../pages/adf/navigationBarPage';
 import { FormCloudDemoPage } from '../../pages/adf/demo-shell/process-services-cloud/cloudFormDemoPage';
 import { checkboxVisibilityForm, multipleCheckboxVisibilityForm } from '../../resources/forms/checkbox-visibility-condition';
+import { multipleVisibilityForm } from '../../resources/forms/multiple-visibility-conditions';
```

Comment:
making that a string and then parsing to JSON makes absolutely no sense as for me. Can you please turn that into a real object and just reference here?